container/image.bzl

```diff
@@ -946,7 +946,7 @@ def container_image(**kwargs):
             layers inside the Docker registry.
 
     This rule references the `@io_bazel_rules_docker//toolchains/docker:toolchain_type`.
-    See [How to use the Docker Toolchain](toolchains/docker/readme.md#how-to-use-the-docker-toolchain) for details.
+    See [How to use the Docker Toolchain](https://github.com/bazelbuild/rules_docker/tree/master/toolchains/docker/readme.md#how-to-use-the-docker-toolchain) for details.
```

Comment:
This would break previewing at a specific PR. Could we change do a similar change except replace `https://github.com/bazelbuild/rules_docker/tree/master/` with `/`? That will allow GitHub to rewrite the urls for a specific branch/pr/commit. This is very useful for looking into the past for docs.

Reply:
@gravypod thanks for your review and suggestions! I made the change and tested it, perfectly works. This will still not work for the documentation published under `docs.aspect.dev. Is this something we should have in mind?

Reply:
Sorry, I fixed and rebased already :upside_down_face: